src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/Helpers.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ServiceProcess.Tests
+{
+    public enum PipeMessageByteCode
+    {
+        Start = 0,
+        Continue = 1,
+        Pause = 2,
+        Stop = 3,
+        OnCustomCommand = 4
+    };
+
+    public static class TaskTimeoutExtensions
+    {
+        public static async Task TimeoutAfter(this Task task, int millisecondsTimeout)
+        {
+            var cts = new CancellationTokenSource();
```

Comment:
@Anipik we should not duplicate code. This should have been pulled from src\Common\tests\System\Threading\Tasks\TaskTimeoutExtensions.cs

Perhaps you can do a follow up PR for that

Reply:
yeah I will change that in a follow up PR